index.html

```diff
@@ -4975,6 +4975,637 @@ <h2 id="AudioWorkerParamDescriptor">
+            constructor.
+          </li>
+        </ol>
+        <section>
+          <h2 id="AudioWorkletNode">
+            The <dfn>AudioWorkletNode</dfn> interface
+          </h2>
+          <p>
+            This interface represents a user-defined
+            <a><code>AudioNode</code></a> which lives on the main <a>control
+            thread</a>. The user can create an <code>AudioWorkletNode</code>
+            from an <a><code>AudioContext</code></a> and such a node can be
+            connected with other built-in <a><code>AudioNode</code></a>s to
+            form an audio graph.
+          </p>
+          <dl title=
+          "[Constructor(BaseAudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode"
+          class="idl">
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The <code>onmessage</code> handler is called whenever the
+                associated <a><code>AudioWorkletProcessor</code></a> posts a
+                message back to the main thread.
+              </p>
+            </dd>
+            <dt>
+              void postMessage(any message, optional
+              sequence&lt;Transferable&gt; transfer)
+            </dt>
+            <dd>
+              <p>
+                <code>postMessage</code> may be called to send a message to the
+                corresponding <a>AudioWorkletProcessor</a> via the algorithm
+                defined by the specification.
+              </p>
+            </dd>
+          </dl>
+          <section>
+            <h2 id="AudioWorkletNodeOptions">
+              <dfn>AudioWorkletNodeOptions</dfn>
+            </h2>
+            <p>
+              The <code>AudioWorkletNodeOptions</code> dictionary can be used
+              for the custom initialization of <a><code>AudioNode</code></a>
+              attributes in the <a><code>AudioWorkletNode</code></a>
+              constructor. Entries in this dictionary whose names correspond to
+              <a><code>AudioParam</code></a>s in the class definition of an
+              <a><code>AudioWorkletProcessor</code></a> are used to initialize
+              the parameter values upon the creation of a node.
+            </p>
+            <dl title="dictionary AudioWorkletNodeOptions" class="idl">
+              <dt>
+                unsigned long numberOfInputs
+              </dt>
+              <dd>
+                Same to <a>numberOfInputs</a> in <a>AudioNode</a>.
+              </dd>
+              <dt>
+                unsigned long numberOfOutputs
+              </dt>
+              <dd>
+                Same to <a>numberOfOutputs</a> in <a>AudioNode</a>.
+              </dd>
+              <dt>
+                unsigned long channelCount
+              </dt>
+              <dd>
+                Same to <a>channelCount</a> in <a>AudioNode</a>.
```

Comment:
Actually, maybe it should inherit from AudioNodeOptions AudioNodeOptions dictionary can't specify any defaults because each has its own set of defaults.  So, for this to work out, I think you need to either make these members be required and/or specify a default value for each.